pkg/errors/errors.go

```diff
@@ -0,0 +1,32 @@
+package errors
+
+import (
+	"github.com/giantswarm/microerror"
+)
+
+var InvalidConfigError = &microerror.Error{
```

Comment:
We shouldn't do that in general. So far all errors were package scoped and we did some huge catch all functions so far. (at least that's what I understood from @tuommaki yesterday). We had a session about that yesterday and there will be some upcoming improvements (https://github.com/giantswarm/giantswarm/issues/7139) but till we have them we need to keep errors inside a package to not create mess.

Reply:
This is a good resource https://github.com/giantswarm/fmt/blob/master/go/errors.md.